arch/Kconfig

```diff
@@ -241,7 +241,7 @@ config CPU_HAS_FPU
 	bool
 	help
 	  This option is enabled when the CPU has hardware floating point
-	  unit.
+	  unit supporting single precision.
```

Comment:
However: is this true for all architectures? (Looks OK for arm)

Reply:
Update: If the symbol FP_DOUBLE_PRECISION is only to be used for ARM compilation options, then I would propose, 
- CPU_HAS_FPU state that "single precision" is implied for ARM only,
- not to select FP_DOUBLE_PRECISION elsewhere, as it will have no effect
- and to get it more accurate, FP_DOUBLE_PRECISION could depend on ARM and also state at the description that it is needed for ARM CPUs to make this differentiation.

Reply:
At the moment it is indeed only needed on ARM, that's why the first version of my PR defined `CPU_HAS_FPU_DP` in `arch/arm/Kconfig`.

I'll modify the code so that it is an ARM only option.